### DISCUSSION TITLE
article/backup-notion-to-github

### DISCUSSION THREAD

**Discussion by giscus** [wanghaoxi3000/darklab-blog#4] [Announcements]
# article/backup-notion-to-github

在开始使用 Notion 编写文档并通过 NextNotion 来发布博客后，整个博客的管理和书写体验感觉流畅了不少，不过整个博客的数据放在 Notion 中，心里多少还是有些担忧，假如以后想再迁移到其他的平台岂不是很麻烦，最好有一种方法可以批量导出 Notion 中的文档为 markdown，这样就可以把数据掌握在自己手里，随时可以迁移，再通过 Git 来管理，每个文档还可以有变更的历史记录，再也没有后顾之忧。

https://darkreunion.tech/article/backup-notion-to-github

**Comment by happyzhangyyds** [comment]
博主大大您好，我在尝试的时候，卡在了第二步和第三步之间，我的blog是通过vercel部署的，这个项目是需要在服务器上才能运行吗，我在本地调试的时候，显示 “未知的写作平台”，希望作者大大能帮忙解决这个问题，非常感谢鸭！

**Reply by wanghaoxi3000** [reply]
我的 blog 也是通过vercel部署的, 备份的过程可以通过 Github Action 来处理, 无需自己的服务器. 提升未知的写作平台感觉可能是配置文件有问题. 可以参考我的 [elog.config.js](https://github.com/wanghaoxi3000/darklab-blog/blob/notion/elog.config.js) 配置文件对比下看看, 或者发下你的配置呢.
